### DISCUSSION TITLE
本地部署安装完成无法登录/注册

### DISCUSSION THREAD

**Discussion by kongcz** [illacloud/illa-builder#1125] [Q&A] [answered]
本地部署，使用root password登录提示无法登录，注册也提示无法注册

查看docker日志发现数据库初始化失败
[error] failed to initialize database, got error failed to connect to `host=localhost user=illa_supervisor database=illa_supervisor`: dial error (dial tcp [::1]:5432: connect: connection refused)

**Comment by owenlongbo** [comment, marked as answer]
亲，可以使用最新的docker看一下是否还有相关问题。
